prog/target.go

```diff
@@ -135,6 +146,14 @@ func (target *Target) lazyInit() {
 	target.initUselessHints()
 	target.initRelatedFields()
 	target.initArch(target)
+	// Fetch the ID for syz_kfuzztest_run, which is built-in for Linux amd64.
+	if target.OS == "linux" && target.Arch == "amd64" {
```

Comment:
Let's make the description itself amd64-only rather than doing it here in the target.go code.

Grep `sys/linux` for `meta arches`.